modules/ROOT/pages/audit-logs.adoc

```diff
@@ -57,7 +58,7 @@ An audit outcome is the final result of an audit event. The final result of audi
 
 == Audit file handler
 
-An audit file handler emits audit data to an on-premises `audit.log` file. By default, the audit file handler writes audit records to an audit log file until the file reaches 20 MB. The audit log file is then archived and a new audit log file is started. The maximum number of archived audit log files, by default, is 100 after which the oldest archived log file is overwritten. To configure the audit file handler, you can configure the `auditFileHandler` element in the `server.xml` file. For more information, see https://draft-openlibertyio.mybluemix.net/docs/ref/feature/#audit-1.0.html[Audit 1.0]. The audit file handler can be customized by specifying any of the config attributes, along with the `auditFileHandler` element. For more information, see link:https://www.openliberty.io/docs/ref/config/#auditFileHandler.html[Default Audit File Handler].
+An audit file handler emits audit data to an on-premises `audit.log` file. By default, the audit file handler writes audit records to an audit log file until the file reaches 20 MB. The audit log file is then archived and a new audit log file is started. The maximum number of archived audit log files, by default, is 100 after which the oldest archived log file is overwritten. To configure the audit file handler, you can configure the `auditFileHandler` element in the `server.xml` file. For more information, see the xref:reference:feature/audit-1.0.adoc[Audit] feature. The audit file handler can be customized by specifying any of the config attributes, along with the `auditFileHandler` element. For more information, see link:https://www.openliberty.io/docs/ref/config/#auditFileHandler.html[Default Audit File Handler].
```

Comment:
I think you can use macros for both the feature and the config element here:

xref:reference:feature/audit-1.0.adoc[Audit] feature
--->
feature:audit[display=Audit] feature

link:https://www.openliberty.io/docs/ref/config/#auditFileHandler.html[Default Audit File Handler]
---->
config:auditFileHandler[Default Audit File Handler]